docs/misc/discrete.md

```diff
@@ -1,39 +1,39 @@
 author: GavinZhengOI
 
-## 简介
+## Introduction
 
-离散化本质上可以看成是一种 [哈希](../string/hash.md) ，其保证数据在哈希以后仍然保持原来的全/偏序关系。
+Discretization can be seen as a type of [hash](../string/hash.md) in essence, which ensures that the data still maintains the original full/partial order relationship after hashing.
 
-通俗地讲就是当有些数据因为本身很大或者类型不支持，自身无法作为数组的下标来方便地处理，而影响最终结果的只有元素之间的相对大小关系时，我们可以将原来的数据按照从大到小编号来处理问题，即离散化。
+In layman's terms, when some data is too large or the data type is not supported, it cannot be easily processed as an array subscript by itself, and only the relative size relationship between the elements affects the final result. We can use the original data according to the serial number from large to small to deal with the problem, that is, discretization.
```

Comment:
> ... not supported, it cannot be easily processed as an array subscript by itself, and only the relative size relationship between
not supported, causing the value itself unable to be array subscripts, and the answer is determined only by the relative size of the elements.